Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -19,6 +19,8 @@
 #include "TRDBase/PadPlane.h"
 #include "TRDBase/PadResponse.h"
 
+#include "DataFormatsTRD/HitType.h"
```

Comment:
Just style, but please reorder the headers. I think DataFormats should precede TRDBase.

Reply:
comment as below